paymentapi/src/main/java/fi/hel/verkkokauppa/payment/paytrail/PaytrailPaymentClient.java

```diff
@@ -90,6 +94,7 @@ public List<PaytrailPaymentMethod> getPaymentMethods(PaytrailPaymentContext cont
     }
 
     private PaytrailPaymentMitChargeSuccessResponse createMitCharge(PaytrailClient client, PaytrailPaymentCreateMitChargeRequest.CreateMitChargePayload payload) throws ExecutionException, InterruptedException {
+        payload.setCallbackDelay(callbackDelay);
```

Comment:
Where is this callBackDelay defined for the payload and where is it used?
I tried looking from paytrail java lib but could not find it in master or a pull request/branch where it would be

Reply:
Good catch, I forgot to push and create a pr of this. There is pr https://github.com/City-of-Helsinki/paytrail-java-lib/pull/27